bindings/python/.cargo/config.toml

```diff
@@ -0,0 +1,14 @@
+[target.x86_64-apple-darwin]
+rustflags = [
+  "-C", "link-arg=-undefined",
+  "-C", "link-arg=dynamic_lookup",
+  "-C", "link-arg=-mmacosx-version-min=10.11",
```

Comment:
Do we still have a link where this is explained ? I can only find the `MACOSX_DEPLOYMENT_TARGET` with my google fu.

(It's not super important)

Reply:
It comes from this thread :
https://users.rust-lang.org/t/compile-rust-binary-for-older-versions-of-mac-osx/38695

Reply:
I think we should leave that here for now. I'll test on next release without that flag to see if it still works.